app/models/concerns/hostext/search.rb

```diff
@@ -119,6 +121,14 @@ def search_by_user(key, operator, value)
         {:conditions => " hosts.id #{opts} " }
       end
 
+      def search_by_usergroup(key, operator, value)
```

Comment:
This  ext_method is not necessary, scoped search will know how to search by the usergroup correctly without it